WordPress/gutenberg
# Bundle new collection of Header and Footer block patterns

## What?

Authored by @beafialho and @kjellr, these block patterns represent a consolidation of recurring headers and footers, simplified in order to work well across themes.

As they go hand in hand with the new site-editing features rolling out this year, the goal is to bundle these in WordPress 6.1 soon, after testing in Gutenberg.

Props beafialho, kjellr

## Testing Instructions

* In the Site Editor, insert the new patterns using either inserter (global inserter, in-between inserter), and verify that they render properly in the canvas, in the inserter preview, and in the front end.
* Alternatively, in the Site Editor canvas, select a Template Part block of type Header or Footer, and in its toolbar click "Replace header/toolbar". In the resulting modal screen, verify that the new patterns are correctly displayed, and that they match the type (i.e. only Footer patterns are offered to replace a Footer template part).
* Please report any instances where the preview of a pattern could be improved (by adding a `viewportWidth` value).

## Issues

- [x] Host [the one image asset](
![](https://headerandfooterpatterns.files.wordpress.com/2022/01/5eca6-header-mountains-scaled-1.jpg)
) in an adequate place like `s.w.org`
- [x] Some patterns are using Navigation blocks with a `ref`, resulting in a broken pattern:
    - [x] header-with-large-font-size
    - [x] centered-logo-in-navigation
    - [x] simple-header-with-background-color
- [x] Rename some incorrectly named files:
    - [x] `centered-header` → `centered-logo-in-navigation` and vice versa
    - [x] `sample-page` → `simple-header`
